packages/documentation/src/stories/foundation/icons/icons.styles.scss

```diff
@@ -1,5 +1,107 @@
-.icons {
+@use '@swisspost/design-system-styles/core' as post;
+
+.icon-search {
+  .icon-search--filters {
+    margin: post.$size-huge auto;
+    max-width: 500px;
+
+    .freetext {
+      position: relative;
+
+      .form-control {
+        padding-right: post.$input-height-lg;
+      }
+      
+      .form-control-reset {
+        position: absolute;
+        top: 0;
+        right: 0;
+        padding: 0;
+        background: none;
+        border: 0 none;
+        
+        post-icon {
+          padding: post.$input-padding-y-lg;
+          width: post.$input-height-lg;
+        }
+      }
+    }
+  }
+
+  .icon-search--results {
```

Comment:
You would achieve a similar result using [.row-cols-*](https://getbootstrap.com/docs/5.3/layout/grid/#row-columns) classes.